src/Scripts/Game/Components/Vehicle/EL_SirenManagerComponent.c

```diff
@@ -187,6 +188,8 @@ class EL_SirenMode
 		new ParamEnum("Mode7", "7")
 	};
 	
+	// Number of modes available, used to reduce network traffic. Increase in powers of 2
+	static const int N_MODES = 8;
```

Comment:
Does this need to be hard coded? Why not just array.Count()? that could likely be made const as well if the array is const so it's only ever evaluated once.

e.g. `const int count = EL_SirenMode.MODE_NAMES.Count()`.

And I think the WriteIntRange is clever enough to know how many bits to use even for odd numbers.